tests/unit/api/search/test_v0.py

```diff
@@ -49,15 +49,16 @@ class SearchTableQueryString(unittest.TestCase):
     def setUp(self) -> None:
         self.mock_table_results = MOCK_TABLE_RESULTS
         self.expected_parsed_table_results = MOCK_PARSED_TABLE_RESULTS
-        self.search_url = local_app.config['SEARCHSERVICE_BASE'] + SEARCH_ENDPOINT
+        self.search_service_url = local_app.config['SEARCHSERVICE_BASE'] + SEARCH_TABLE_ENDPOINT
+        self.fe_flask_endpoint = '/api/search/v0/table'
```

Comment:
The tests are split between user search and table search. I think we clobbered some of the user searches.

Reply:
Is this a misunderstanding of the diff? The user searches are in a separate test class with its own `setUp()` that uses `/api/search/v0/user`.

Reply:
I missed the user test setup on line 222.